src/scaffoldmaker/meshtypes/meshtype_3d_tubenetwork1.py

```diff
@@ -98,33 +37,33 @@ def getDefaultOptions(cls, parameterSetName="Default"):
             "Network layout": ScaffoldPackage(MeshType_1d_network_layout1,
                                               {"scaffoldSettings": {"Define inner coordinates": True}},
                                               defaultParameterSetName=parameterSetName),
-            "Elements count around": 8,
-            "Elements count through wall": 1,
-            "Annotation elements counts around": [0],
+            "Number of elements around": 8,
+            "Number of elements through shell": 1,
+            "Annotation numbers of elements around": [0],
             "Target element density along longest segment": 4.0,
-            "Use linear through wall": False,
+            "Use linear through shell": False,
             "Show trim surfaces": False,
             "Core": False,
-            'Number of elements across core major': 4,
-            'Number of elements across core transition': 1,
-            "Annotation elements counts across major": [0]
+            "Number of elements across core box minor": 2,
+            "Number of elements across core transition": 1,
+            "Annotation numbers of elements across core box minor": [0]
```

Comment:
It's for future use: it could be used for disconnected networks, and also odd minor numbers can specify a half phase difference when core is not used, as that's needed when core is used.